libretro-common/samples/core_options/example_translation/translation scripts/.github/workflows/crowdin_translate.yml

```diff
@@ -1,21 +1,23 @@
-# Recreate libretro_core_options_intl.h using translations form Crowdin
+# Download translations form Crowdin & Recreate libretro_core_options_intl.h
 
 name: Crowdin Translation Integration
 
 on:
-  push:
-    branches:
-      - master
-    paths:
-      - 'intl/*/*'
+  schedule:
+    - cron: '0 0 * * *'  # every day at midnight
```

Comment:
I suggest to add one warning here:

> please use a different & random schedule time to avoid all repos stressing GitHub and Crowdin at the same time...

Reply:
You think so, too, huh? I _was_ worried this might lead to problems.